frontend/.env.example

```
@@ -9,10 +9,16 @@ APP_URL=
 NEXT_PUBLIC_POSTHOG_KEY=
+POSTHOG_API_KEY=
 NEXT_PUBLIC_POSTHOG_HOST=
 NEXT_PUBLIC_APP_ENV=local
 
-NEXT_PUBLIC_STRAPI_API_URL=http://localhost:1337
-STRAPI_ACCESS_TOKEN= #empty if using docker compose
-
-NEXT_PUBLIC_APP_ENV=test
\ No newline at end of file
+STRAPI_API_URL=
+STRAPI_ACCESS_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The STRAPI_ACCESS_TOKEN key should go before the STRAPI_API_URL key

(UnorderedKey)